src/main/java/core/basesyntax/Area.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface Area {
+    void findArea();
```

Comment:
it would be better to return our area, so return type can be `double` for example